Src/library/docking/implementation/docking/sd_docking_manager_agents.e

```diff
@@ -208,6 +209,57 @@ feature  -- Agents
 			l_docking_manager.tool_bar_manager.on_resize (a_x, a_y, l_docking_manager.internal_viewport.width, l_docking_manager.internal_viewport.height, a_force)
 		end
 
+
+	on_dpi_change_resize (a_dpi,a_x: INTEGER; a_y: INTEGER; a_width: INTEGER; a_height: INTEGER; a_force: BOOLEAN)
+			-- Handle resize zone event. Resize all the widgets in fixed_area (EV_FIXED)
+		require
+			not_destroyed: not is_destroyed
+		local
+			l_width, l_height: INTEGER
+			l_main_container: SD_MULTI_DOCK_AREA
+			l_docking_manager: like docking_manager
+		do
+			debug ("docking")
+				io.put_string ("%N SD_DOCKING_MANAGER on_resize ~~~~~~~~~~~~~~~~~~~~")
```

Comment:
in manifest string : on_resize or on_dpi_change_resize ?

Reply:
and this one ?

Reply:
Done